Mesos/start-windows-build.ps1

```diff
@@ -35,7 +35,7 @@ function Install-Prerequisites {
             'install_args' = @("/SILENT")
             'install_dir' = $GIT_DIR
         }
-        'cmake'= @{
+        'go'= @{
```

Comment:
Why does exactly you do this change ?

Reply:
It looks like a bad merge was causing those extra files and this unintended change to be included. I am trying to address this.